readme.md

```diff
@@ -326,6 +326,7 @@ Thanks for contributing! 🦋🙌
 - [](# "easy-toggle-files") [Enables toggling file diffs by clicking on their header bar.](https://user-images.githubusercontent.com/47531779/99855419-be173e00-2b7e-11eb-9a55-0f6251aeb0ef.gif)
 - [](# "same-branch-author-commits") [Preserves current branch and path when viewing all commits by an author.](https://user-images.githubusercontent.com/44045911/148764372-ee443213-e61a-4227-9219-0ee54ed832e8.png)
 - [](# "easy-toggle-commit-messages") [Enables toggling commit messages by clicking on the commit box.](https://user-images.githubusercontent.com/1402241/152121837-ca13bf8a-9b7f-4517-8e8d-b58bb135523b.gif)
+- [](# "sticky-file-header") [Enables sticky file header on commits and compare pages.](https://user-images.githubusercontent.com/44045911/156827766-283e8388-7053-418f-a38b-16f615c6af4a.gif)
```

Comment:
```suggestion
- [](# "sticky-file-header") [Enables sticky file headers on commit and compare pages.](https://user-images.githubusercontent.com/44045911/156827766-283e8388-7053-418f-a38b-16f615c6af4a.gif)
```

Reply:
🥲 actually this should not be in the README anymore